app/src/main/java/swati4star/createpdf/util/WatermarkUtils.java

```diff
@@ -116,9 +117,11 @@ public void afterTextChanged(Editable input) {
                         Color.blue(colorPickerInput.getColor()),
                         Color.alpha(colorPickerInput.getColor())
                 )));
-                createWatermark(path);
+                String output = createWatermark(path);
```

Comment:
@piedcipher 
Just a little change 'output' to something related the function is returning.
I think it is will be path if so change it to `filePath` or something

Reply:
Done.